### ISSUE TITLE
previousErrorHandler number of arguments

### ISSUE THREAD

**Issue by hhoechtl** [getsentry/sentry-php#891]
```
app/vendor/sentry/sentry/src/ErrorHandler.php on line 386 and exactly 5 expected in /app/vendor/shopware/shopware/engine/Shopware/Plugins/Default/Core/ErrorHandler/Bootstrap.php on line 183
```

Originally caused by the following error:
```
The Sentry\State\Hub::getCurrent() method is deprecated since version 2.2 and will be removed in 3.0. Use SentrySdk::getCurrentHub() instead.
```

The `previousErrorHandler` registered accepts 5 arguments. The fifth argument is the `$errcontext` according to https://www.php.net/manual/de/function.set-error-handler.php.

I've fixed the deprecated call for now, but the errorHandler should in general deal with the fifth optional argument.

**Comment by Jean85** [comment]
I'm not sure if I understood the source of this error... The fifth parameter is optional AND deprecated as of PHP 7.2, so we cannot use it. Not having it in the function signature shouldn't be an issue.

**Comment by hhoechtl** [comment]
The application I'm in has an already registered error_handler. It's a closure with exactly 5 arguments.
Sentry calls the `previousErrorHandler` with only 4 which is the problem. I know the 5th argument is deprecated but obviously it must be present if the closure requires it.

**Comment by Jean85** [comment]
To my knowledge, PHP doesn't complain about invoking a function with more arguments, see https://3v4l.org/jf5va

So, you're probably doing something additional on your side to trigger this issue.

**Comment by hhoechtl** [comment]
I don't have a patch, I just fixed all errors in my application and that deprecated Sentry SDK call, so I don't have a problem atm.

But I found this: https://www.php.net/manual/de/migration71.incompatible.php

**Comment by ste93cry** [comment]
>I don't have a patch

What I meant was if you could be so kind to submit a PR to fix the problem in our library (the fact that we don't pass the 5th argument). The problem you had will came back as soon as Sentry logs an error since it will again call the previous error handler without the `$errcontext` :wink:

**Comment by hhoechtl** [comment]
I don't think https://github.com/getsentry/sentry-php/commit/72c95d1b8c5ae7e1141d589990aa62780259ad88 is enough, the problem would be 

```php
if (null !== $this->previousErrorHandler) {
            return false !== \call_user_func($this->previousErrorHandler, $level, $message, $file, $line);
        }
```

**Comment by ste93cry** [comment]
Closing as it has been solved in #892
